nbs/api/cli.ipynb

```diff
@@ -172,6 +172,25 @@
     "                  \"Use a token with the correction permissions or perform these steps manually.\")"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            tag = _nbdev_template_release((1,1)).tag_name
</pre><p>I wonder if there is a way to make it such that we can remind ourselves that we must change this line of code when we make a breaking change in the template repository.  Surely, we will forget to this otherwise?  </p><p> </p><p>Any ideas?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1094/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Good point. I don't have a good answer yet, best I can think of is an e2e test for `nbdev_new` that (hopefully) fails if we forget to bump

Reply:
@jph00 I'm curious to hear your take on this